packages/core/src/__tests__/useFocusEffect.test.tsx

```diff
@@ -45,17 +45,23 @@ it('runs focus effect on focus change', () => {
   expect(focusEffect).not.toBeCalled();
   expect(focusEffectCleanup).not.toBeCalled();
 
-  act(() => navigation.current.navigate('second'));
+  act(() => {
+    navigation.current.navigate('second');
+  });
 
   expect(focusEffect).toBeCalledTimes(1);
   expect(focusEffectCleanup).not.toBeCalled();
 
-  act(() => navigation.current.navigate('third'));
+  act(() => {
+    navigation.current.navigate('third');
```

Comment:
I think we should revert this change because it caught an issue where we were leaking internal return value as the return value of `dispatch`